### PR TITLE
[Log] Commit log timing moved

### PR DESCRIPTION
**Pull request by hyunsooda**
## Proposed changes

Minor change: Push a log once the commit work is successfully done

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: blockchain/blockchain.go [hunk 1]
```diff
@@ -1351,10 +1351,10 @@ func (bc *BlockChain) writeStateTrie(block *types.Block, state *state.StateDB) e
 		}
 
 		if isCommitTrieRequired(bc, block.NumberU64()) {
-			logger.Trace("Commit the state trie into the disk", "blocknum", block.NumberU64())
 			if err := trieDB.Commit(block.Header().Root, true, block.NumberU64()); err != nil {
 				return err
 			}
+			logger.Trace("Commit the state trie into the disk", "blocknum", block.NumberU64())
```
Comment (hyeonLewis) [inline blockchain/blockchain.go:1357]: How about using `Committed the ~`?

Comment (hyunsooda) [reply blockchain/blockchain.go:1357]: Good suggestion. Before making a change, let's hold off for a bit and gather input from others to ensure consensus.

Comment (yoomee1313) [reply blockchain/blockchain.go:1357]: I agree with the above comment as the timing of the log output has changed to after the commit is done.

Comment (yoomee1313) [reply blockchain/blockchain.go:1357]: In addition, it's trace log, so it won't affect monitoring service. Could you proceed this comment?

Comment (hyunsooda) [reply blockchain/blockchain.go]: Thank you all. Updated.

**Review by ian0371 — APPROVED** [review]

**Review by hyeonLewis — APPROVED** [review]

**Review by yoomee1313 — APPROVED** [review]

**Pull request merged**